tests/e2e/translations/message-format.spec.ts.js

```diff
@@ -0,0 +1,51 @@
+const utils = require('../../utils');
+const helper = require('../../helper');
+const commonElements = require('../../page-objects/common/common.po.js');
```

Comment:
```suggestion
const commonElements = require('../../page-objects/common/common.po');
```

Reply:
Yea, the IDE autocompletes these for me. I think it may have been confused by the .ts.js extension, because it usually does the right thing :D